integrations/tensorflow/iree_tf_compiler/TFL/test/lower_global_tensors.mlir

```diff
@@ -100,6 +100,36 @@ module {
 
 // -----
 
+// CHECK-LABEL: module {
+module {
+  // func @readAssignQuant
+  func @readAssignQuant(%arg0: tensor<2x3x!quant.uniform<i8:f32, 0.1:2>>) -> (tensor<2x3x!quant.uniform<i8:f32, 0.1:2>>) {
```

Comment:
CHECK-LABEL is theoretically supposed to be unique in the file (https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-label-directive)

And also having the explicit module shouldn't be necessary?

```suggestion
  // CHECK-LABEL: func @readAssignQuant(
  func @readAssignQuant(%arg0: tensor<2x3x!quant.uniform<i8:f32, 0.1:2>>) -> (tensor<2x3x!quant.uniform<i8:f32, 0.1:2>>) {
```

Reply:
Done with the tests reworked a bit. Global variables just make check-tests a bit of a pain.